content/docs/implementation-notes.md

```diff
@@ -715,82 +714,82 @@ We collect DOM operations on children in a list so we can execute them in batch:
     if (!Array.isArray(nextChildren)) {
       nextChildren = [nextChildren];
     }
-    // These are arrays of internal instances:
+    // Ini adalah senarai dari instance internal:
     var prevRenderedChildren = this.renderedChildren;
     var nextRenderedChildren = [];
 
-    // As we iterate over children, we will add operations to the array.
+    // Saat kita beralih ke children, kita akan menambahkan operasi ke senarai.
     var operationQueue = [];
 
-    // Note: the section below is extremely simplified!
-    // It doesn't handle reorders, children with holes, or keys.
-    // It only exists to illustrate the overall flow, not the specifics.
+    // Catatan: bagian di bawah ini sangat disederhanakan!
+    // Itu tidak menangani pemesanan ulang, *children* dengan *holes*, atau kunci.
```

Comment:
```suggestion
    // Itu tidak menangani pemesanan ulang, *children* dengan *holes*, atau *key*.
```

"key" doesn't need t be translated, as per the [glossary](https://github.com/reactjs/id.reactjs.org/wiki/Glosarium#k).

Reply:
Oops, sorry I missing that